neuro_extras/main.py

```diff
@@ -91,7 +91,11 @@ def _create_builder_container(
         context_uri: URL,
         dockerfile_path: str,
         image_ref: str,
+        build_args: Sequence[str] = [],
     ) -> neuro_api.Container:
+        command = f"--dockerfile={dockerfile_path} --destination={image_ref}"
+        if len(build_args) > 0:
+            command += " " + " ".join([f"--build-arg {arg}" for arg in build_args])
```

Comment:
```suggestion
        if build_args:
            command += "".join([f" --build-arg {arg}" for arg in build_args])
```
slightly simplified.